packages/amplify-codegen/src/commands/models.js

```diff
@@ -44,6 +45,85 @@ const readNumericFeatureFlag = key => {
   }
 };
 
+/**
+ * Returns the version number of package installed
+ * @param {string} target language target
+ * @returns
+ */
+const readDependencyVersionInstalled = (dependency, target) => {
+  // TODO: other platforms and error handling
+  const projectPath = pathManager.findProjectRoot();
+  switch (target) {
```

Comment:
If we're going ahead with this, it has to be consistent across platforms. We should be able to answer the question, is this possible for native platforms?